contrib/win32/win32compat/misc.c

```diff
@@ -314,22 +363,35 @@ spawn_child(char* cmd, int in, int out, int err, DWORD flags) {
 	si.dwFlags = STARTF_USESTDHANDLES;
 
 	b = CreateProcessW(NULL, cmd_utf16, NULL, NULL, TRUE, flags, NULL, NULL, &si, &pi);
+	GOTO_CLEANUP_IF(!b, ENOEXEC);
+	CloseHandle(pi.hThread);
```

Comment:
at line 365, pi is passed as an argument to the CreateProcessW() so the newly create process should close the handle?